application/backup.sh

```diff
@@ -2,87 +2,41 @@
 
 # Initialize logging with timestamp
 log() {
-    echo "[$(date '+%Y-%m-%d %H:%M:%S')] $1"
+    local level="${1:-INFO}"
```

Comment:
The `:-INFO` part won't work quite right. If `log` is passed only one parameter, it will be assigned to $1, which would be interpreted as the message. Since bash doesn't have named parameters, you will need to ensure that `log` receives two parameters in every use.